ramls/edge-rtac.raml

```diff
@@ -9,7 +9,30 @@ documentation:
 
 schemas:
   holdings: !include holdings.xsd
+  batchHoldings: !include batch-holdings.xsd
+  errors: !include raml-util/schemas/error.schema
 
+/rtac:
+  displayName: RTAC
+  get:
+    description: Batch RTAC for the specified ids
+    responses:
+      200:
+        description: "Success"
+        body:
+          application/xml:
+            type: batchHoldings
+    queryParameters:
+      mms_id:
```

Comment:
Has any thought been given to how many ids we can support (you'll eventually run up against url max lengths).  It might be reasonable to explicitly cap it here by setting maxLength and adding a note to the description

Reply:
According to the PO, it will be not more then 50.